example/basic_extract.dart

```diff
@@ -3,7 +3,7 @@ import 'package:metadata_fetch/metadata_fetch.dart';
 void main() async {
   var data = await extract('https://flutter.dev'); // returns a Metadata object
   print(data); // Metadata.toString()
-  print(data.title); // Metadata.title
-  print(data.toMap()); // converts Metadata to map
-  print(data.toJson()); // converts Metadata to JSON
+  print(data?.title); // Metadata.title
```

Comment:
Question mark here is useless, because Metadata is non-nullable. I just forgot to remove.